src/cases/routes/login-callback.route.js

```diff
@@ -35,23 +39,36 @@ export const loginCallbackRoute = {
       strategy: "msEntraId",
     },
   },
-  handler: async function (request, h) {
-    if (request.auth.isAuthenticated === false) return notAuthorised();
-    const roles = getRoles(request.auth.artifacts);
-    const authorised = validateRoles(roles);
-    if (!authorised) return notAuthorised();
-    const next = getNextDestination(request.auth.credentials);
+  // eslint-disable-next-line complexity
```

Comment:
The complexity will decrease when the login use case is added - user creation, update etc